https://github.com/util-linux/util-linux/issues/2119
https://github.com/util-linux/util-linux/commit/bccae5d85831c4cfa6b5d147acec739bd3c148b9

From bccae5d85831c4cfa6b5d147acec739bd3c148b9 Mon Sep 17 00:00:00 2001
From: Karel Zak <kzak@redhat.com>
Date: Fri, 17 Mar 2023 13:11:50 +0100
Subject: [PATCH] build-sys: fix libblkid fuzz sample test if-endif

All tests have to be covered by BUILD_LIBBLKID_TESTS otherwise
proper CFLAGS ($blkid_tests_cflags) are undefined.

Fixes: https://github.com/util-linux/util-linux/issues/2119
Signed-off-by: Karel Zak <kzak@redhat.com>
--- a/libblkid/src/Makemodule.am
+++ b/libblkid/src/Makemodule.am
@@ -224,8 +224,6 @@ test_blkid_fuzz_LDFLAGS = $(blkid_tests_ldflags) -lpthread
 test_blkid_fuzz_LDADD = $(blkid_tests_ldadd) $(LIB_FUZZING_ENGINE)
 endif
 
-endif # BUILD_LIBBLKID_TESTS
-
 check_PROGRAMS += test_blkid_fuzz_sample
 
 test_blkid_fuzz_sample_SOURCES = libblkid/src/fuzz.c
@@ -234,6 +232,7 @@ test_blkid_fuzz_sample_CFLAGS = $(blkid_tests_cflags)
 test_blkid_fuzz_sample_LDFLAGS = $(blkid_tests_ldflags)
 test_blkid_fuzz_sample_LDADD = $(blkid_tests_ldadd)
 
+endif # BUILD_LIBBLKID_TESTS
 
 # move lib from $(usrlib_execdir) to $(libdir) if needed
 install-exec-hook-libblkid:

